main.py

```diff
@@ -1,7 +1,67 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(
+        name="Western",
+    )
+    Genre.objects.create(
+        name="Action",
+    )
+    Genre.objects.create(
+        name="Dramma",
+    )
+    Actor.objects.create(
+        first_name="George",
+        last_name="Klooney"
+    )
+    Actor.objects.create(
+        first_name="Kianu",
+        last_name="Reaves"
```

Comment:
There is a typo in the first name 'Kianu' and the last name 'Reaves'. They should be 'Keanu' and 'Reeves', respectively. This is corrected later in the code, but it's better to insert the correct names initially.